### PR TITLE
Add onboarding instructions for Microsoft Team members

### PR DESCRIPTION
**Pull request by richlander**
/cc @terrajobst @karelz

### CODE REVIEW

File: Documentation/microsoft-team.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+If you are new to Microsoft, you will need to onboard into various GitHub projects in order to get your work done.
+
+## Getting Permissions
+
+There are two main teams to join to get access to Microsoft .NET-related projects.
+
+* [microsoft team - dotnet org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
+* [dotnet team - microsoft org](https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/)
+
```
Comment (karelz) [inline Documentation/microsoft-team.md]: Can we add ask to:
1. Make membership public information (they will show as badges on your profile).
2. Make sure either GH name matches GAL name, or make your MS email public -- motivation: other MS employees can recognize you & find you internally if they need to follow up.
3. Add picture (showing face to community for better communication).

Comment (Petermarcu) [reply Documentation/microsoft-team.md]: Add picture for sure.

**Comment by Petermarcu** [comment]
LGTM

File: Documentation/microsoft-team.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Microsoft Team Onboarding
+
+If you are new to Microsoft, you will need to onboard into various GitHub projects in order to get your work done.
+
+## Getting Permissions
+
+There are two main teams to join to get access to Microsoft .NET-related projects.
```
Comment (terrajobst) [inline Documentation/microsoft-team.md]: We should mention linking the [corpnet account to GitHub](https://repos.opensource.microsoft.com/link)

File: Documentation/microsoft-team.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+* [microsoft team - dotnet org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
+* [dotnet team - microsoft org](https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/)
+
+Once your permission requests are approved, you should have write permissions to [.NET Core Repos](https://github.com/dotnet/core/blob/master/Documentation/core-repos.md). Everyone using these repos should also be able to @mention you, too.
+
```
Comment (terrajobst) [inline Documentation/microsoft-team.md]: Maybe add a section for tools? I suggest the following:

* [Chrome extension to see who is an FTE](https://repos.opensource.microsoft.com/settings/security/tokens/extension)
* [Chrome extension for CodeFlow on GitHub](https://chrome.google.com/webstore/detail/codeflow/aphnoipocoffpdafmiidfmaiadhilelm)

File: Documentation/microsoft-team.md (hunk 1 above)
Comment (akoeplinger) [reply Documentation/microsoft-team.md]: Just to make sure: for 2. by GH name you mean the full name, not the alias right?

**Pull request merged**